dbws/org.eclipse.persistence.dbws.maven/pom.xml

```diff
@@ -0,0 +1,231 @@
+                    </execution>
+                    <execution>
+                        <id>stop-derby</id>
+                        <phase>post-integration-test</phase>
+                        <goals>
+                            <goal>stop</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <!--This plugin sql-maven-plugin must be after maven-surefire-plugin to call dbteardown*.sql scripts after tests-->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>sql-maven-plugin</artifactId>
+                <configuration>
+                    <!--all executions are ignored if -Dmaven.test.skip=true-->
+                    <skip>${test-skip-dbws}</skip>
+                </configuration>
+                <executions>
+                    <!-- create db tables after test -->
+                    <execution>
+                        <id>create-tables-before-test</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>execute</goal>
+                        </goals>
+                        <configuration>
+                            <autocommit>true</autocommit>
+                            <delimiter>|</delimiter>
+                            <onError>continue</onError>
+                            <srcFiles>
+                                <srcFile>${integration.test.directory}/resources/sql/dbsetup_keymappings.sql</srcFile>
+                                <srcFile>${integration.test.directory}/resources/sql/dbsetup_relationships.sql</srcFile>
+                            </srcFiles>
+                        </configuration>
+                    </execution>
+                    <!-- drop db tables after test -->
+                    <execution>
+                        <id>drop-tables-after-test</id>
+                        <phase>post-integration-test</phase>
+                        <goals>
+                            <goal>execute</goal>
+                        </goals>
+                        <configuration>
+                            <autocommit>true</autocommit>
+                            <delimiter>|</delimiter>
+                            <onError>continue</onError>
+                            <srcFiles>
+                                <srcFile>${integration.test.directory}/resources/sql/dbteardown_keymappings.sql</srcFile>
+                                <srcFile>${integration.test.directory}/resources/sql/dbteardown_relationships.sql</srcFile>
+                            </srcFiles>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>java11-dependencies</id>
+            <activation>
+                <jdk>(1.11,</jdk>
```

Comment:
this should be '[11,)' instead (or '[9,)'...)

Reply:
OK fixed.